### DISCUSSION TITLE
Why plotman starts only 1 plotting task in 1 round (30min), but I have 2 separate temp folders (disks)?

### DISCUSSION THREAD

**Discussion by Fumo696** [ericaltendorf/plotman#678] [Q&A]
Hi there! 

Subj. 

**How to push it to start 2 tasks in parallel, one for each folder?**

My config: 

> tmp:
>                  - /temp_01
>                  - /temp_02
> dst:
>               - /plots       
> 
> scheduling:
>         tmpdir_stagger_phase_major: 2
>         tmpdir_stagger_phase_minor: 1
>         tmpdir_stagger_phase_limit: 6
>         tmpdir_max_jobs: 5
>         global_max_jobs: 12
>         global_stagger_m: 30
>         polling_time_s: 30
> 
> plotting:
>         k: 32
>         e: False
>         n_threads: 4
>         n_buckets: 128
>         job_buffer: 5000

**Comment by CrossBread** [comment]
You have the global stagger time set to 30 minutes:

global_stagger_m: 30

**Reply by Fumo696** [reply]
Its 30 already. 

My question is: **Any possibility to push plots to start 2 in parallel, one for each folder?**
